rest/taskana-rest-spring/src/main/java/pro/taskana/rest/common/models/PagedResources.java

```diff
@@ -61,8 +61,8 @@ public PagedResources(Collection<T> content, PageMetadata metadata, Iterable<Lin
   @JsonProperty("page")
   public PageMetadata getMetadata() {
     if (Objects.isNull(metadata)) {
-      Collection<T> content = getContent();
-      return new PageMetadata(content.size(), 0, content.size());
+      Collection<T> contentCollection = getContent();
+      return new PageMetadata(contentCollection.size(), 0, contentCollection.size());
     }
     return metadata;
   }
```

Comment:
This is an artifact for taskana-history, right?

Reply:
This is a suggestion by sonarcloud to make sure the field with the same name isn't hidden.